packages/mask/src/plugins/EVM/services/Asset/index.ts

```diff
@@ -94,6 +98,8 @@ export async function getListings(options: ListOption) {
             return Rarible.getListings(address, tokenId, { chainId })
         case NonFungibleAssetProvider.ZORA:
             return Zora.getListings(address, tokenId)
+        case NonFungibleAssetProvider.LOOKSRARE:
+            return Zora.getListings(address, tokenId)
```

Comment:
?

Reply:
I will change it when I implement the Looksrare getListings